src/yadism/coefficient_functions/asy/kernels.py

```diff
@@ -0,0 +1,191 @@
+"""Collections implementing the asymptotic components for FONLL prescription.
+
+This is strictly following the original reference :cite:`forte-fonll`, and
```

Comment:
Here we need to fix some docstrings.
At least the more confusing ie the one citing the original fonll paper.